backend/dotenv.dev.example

```diff
@@ -20,3 +20,6 @@ OIDC_REDIRECT_ALLOWED_HOSTS=localhost:5173
 PLAYWRIGHT_BROWSER=firefox
 PLAYWRIGHT_HEADLESS=no
 
+# For recording VCR cassettes
+# VCR_RECORD_MODE=new_episodes
```

Comment:
My workflow is to `rm` cassettes to re-record, and leave this on `once` in development.
recording completely fresh cassettes has the advantage that if there's no change in our code, the order of episodes is the same, and you have a nice diff of what changed in the upstream responses.